src/ProgressOnderwijsUtils/Extensions/StringExtensions.cs

```diff
@@ -34,30 +33,29 @@ public static string NullIfWhiteSpace([CanBeNull] this string str)
         /// <summary>
         /// HTML-alike whitespace collapsing of this string; however, this method also trims.
         /// </summary>
-        [NotNull]
+        [JetBrains.Annotations.NotNull]

```

Comment:
waarom willen we deze houden?

Reply:
kan wel weg denk ik

Reply:
Die hele method lijkt me dubieus, als we al CollapseWhitespace en Trim hebben.